src/SFA.DAS.Authorization/AuthorizationService.cs

```diff
@@ -58,7 +59,8 @@ public async Task<AuthorizationResult> GetAuthorizationResultAsync(params string
 
         public bool IsAuthorized(params string[] options)
         {
-            return IsAuthorizedAsync(options).GetAwaiter().GetResult();
+            return Task.Run(() => IsAuthorizedAsync(options)).GetAwaiter().GetResult();
+            //return IsAuthorizedAsync(options).GetAwaiter().GetResult();
```

Comment:
As above, If we're happy with the task.Run wrapper we could lose this commented line